apps/api/src/app/storage/usecases/get-signed-url/get-signed-url.usecase.ts

```diff
@@ -3,18 +3,34 @@ import * as hat from 'hat';
 import { StorageService } from '../../../shared/services/storage/storage.service';
 import { UploadUrlResponse } from '../../dtos/upload-url-response.dto';
 import { GetSignedUrlCommand } from './get-signed-url.command';
+import { GetOrganization } from '../../../organization/usecases/get-organization/get-organization.usecase';
 
 const mimeTypes = {
   jpeg: 'image/jpeg',
   png: 'image/png',
+  jpg: 'image/jpeg',
 };
 
 @Injectable()
 export class GetSignedUrl {
-  constructor(private storageService: StorageService) {}
+  constructor(private storageService: StorageService, private getOrganization: GetOrganization) {}
 
   async execute(command: GetSignedUrlCommand): Promise<UploadUrlResponse> {
-    const path = `${command.organizationId}/${command.environmentId}/${hat()}.${command.extension}`;
+    const organization = await this.getOrganization.execute({ id: command.organizationId, userId: command.userId });
+
+    let path = `${command.organizationId}/${command.environmentId}/${hat()}.${command.extension}`;
+
+    if (organization.branding.logo) {
+      const currentImageName = organization.branding.logo.split('/').pop().split('.');
+      const currentImagePrefix = currentImageName.shift();
```

Comment:
Hi @devblin 
You can use this helper method to get file exension.
https://github.com/novuhq/novu/blob/main/apps/api/src/app/shared/services/helper/helper.service.ts#L13